capture/genBitmaps.js

```diff
@@ -12,12 +11,25 @@ if (!config.paths) {
   config.paths = {};
 }
 
+String.prototype.hashCode = function () {
```

Comment:
Well it was "lented" from here: http://werxltd.com/wp/2010/05/13/javascript-implementation-of-javas-string-hashcode-method/ ;)